Source/Fuse.Models/FuseJS/Internal/Model.js

```diff
@@ -0,0 +1,489 @@
+var TreeObservable = require("FuseJS/TreeObservable")
+
+require("Polyfills/Window");
+require("./zone.min")
+require("./ZonePatches");
+
+var rootZone = Zone.current;
+
+function shouldEmitProperty(key) {
+	return !key.startsWith("__fuse_");
```

Comment:
Perhaps it would be a good idea to say that any property starting with double underscores are omitted? That way, the user can intentionally hide implementation details as well, which seems to me like a good idea.

Reply:
There's supposed to be an "underscore" convention to omit variables already (single underscore).  If that's not a feature then we need to add (can be done later). Otherwise our state will include way too much stuff.